OsmAnd-java/src/main/java/net/osmand/data/Amenity.java

```diff
@@ -11,23 +9,13 @@
 import net.osmand.osm.PoiType;
 import net.osmand.util.Algorithms;
 import net.osmand.util.MapUtils;
-
 import org.json.JSONObject;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
 import java.util.Map.Entry;
-import java.util.Set;
-import java.util.TreeSet;
 
-import gnu.trove.list.array.TIntArrayList;
+import static net.osmand.gpx.GPXUtilities.AMENITY_PREFIX;
```

Comment:
Constantly changing... however no imports changed actually, java.util.* not by default old guidelines